Octokit.Tests.Integration/Clients/RepositoryCollaboratorClientTests.cs

```diff
@@ -136,4 +136,26 @@ public class TheIsCollaboratorMethod
             }
         }
     }
+
+    public class TheInviteMethod
+    {
+        [IntegrationTest]
+        public async Task CanInviteNewCollaborator()
+        {
+            var github = Helper.GetAuthenticatedClient();
+            var repoName = Helper.MakeNameWithTimestamp("public-repo");
+
+            using (var context = await github.CreateRepositoryContext(new NewRepository(repoName)))
+            {
+                var fixture = github.Repository.Collaborator;
+                var permission = new CollaboratorRequest();
+
+                // invite a collaborator
+                var response = await fixture.Invite(context.RepositoryOwner, context.RepositoryName, "maddin2016", permission);
```

Comment:
@ryangribble there is some problem with deleting the invitation
```c#
using (var context = await github.CreateRepositoryContext(new NewRepository(repoName)))
            {
                var fixture = github.Repository.Collaborator;
                var permission = new CollaboratorRequest();

                // invite a collaborator
                var response = await fixture.Invite(context.RepositoryOwner, context.RepositoryName, "lrz-hal", permission);

                Assert.Equal("lrz-hal", response.Invitee.Login);
                Assert.Equal(InvitationPermissionType.Write, response.Permissions);

                var deleteInvite = await github.Repository.Invitation.Delete(context.Repository.Id, response.Id);
            }
```
The problem is when `fixture.invite` is calling, in the same moment `context` is trying to delete the repo. So i cannot delete the invitation, because i don't reach that call!!??